scripts/book.js

```diff
@@ -0,0 +1,112 @@
+/* eslint-disable max-classes-per-file */
+class Book {
+  constructor(author, title, nPages, read) {
+    this.author = author;
+    this.title = title;
+    this.nPages = nPages;
+    this.read = read;
+    this.id = title.slice(0, 3).toUpperCase() + nPages;
+  }
+}
+
+function change(e) {
+  const elem = document.getElementById(e.target.id);
+  if (elem.innerHTML === 'Read') elem.innerHTML = 'Unread';
+  else elem.innerHTML = 'Read';
+}
```

Comment:
- [ ] Please convert this to an es6 arrow function syntax. It's important to keep up to date on present trends.